libr/debug/dreg.c

```diff
@@ -120,8 +120,10 @@ R_API int r_debug_reg_list(RDebug *dbg, int type, int size, int rad, const char
 	if (dbg->regcols) {
 		cols = dbg->regcols;
 	}
-	if (rad == 'j') {
-		dbg->cb_printf ("{");
+	PJ *pj = NULL;
+	if (rad == 'j' || rad == 'J') {
+		pj = pj_new ();
```

Comment:
i think that we can introduce some custom nullchks to avoid having them on release builds. What do you think @ret2libc ? i think that small allocations like this shuold not fail except if things are very wrong somewhere, so maybe we can shortten some more paths by asuming fixed small allocations wont fail